waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -40,7 +44,7 @@ migrateDev projectDir = do
   --   we are using `script` to trick Prisma into thinking it is running in TTY (interactively).
 
   -- NOTE(martin): For this to work on Mac, filepath in the list below must be as it is now - not wrapped in any quotes.
-  let npxPrismaMigrateCmd = npxPrismaCmd ++ ["migrate", "dev", "--schema", SP.toFilePath schemaFile]
+  let npxPrismaMigrateCmd = npxPrismaCmd ++ ["migrate", "dev", "--schema", SP.toFilePath schemaFile] ++ migrationArgs
```

Comment:
Yeah `additionalMigrationsArgs` sounds good, thnx. 👍🏻 

Unsure what you mean by redefine schema though?

Reply:
Ah sorry, that was badly explained: Waht I meant to say is, if they provide `--schema` argument under `additionalMigrationArgs`. They could do that and mess up our `--schema` argument. But ok, probably best not to worry about it for now.

Reply:
Ah ok cool, good point to at least be aware of... it may be the case that the flag would get swallowed before making it to us as well, but something to consider if we ever need to.